rails-tips/README.md

```diff
@@ -33,6 +33,14 @@ Rails tips are fun tid-bits of info that aren't best practices or opinionated.
   end
   ```
 
+- On ActiveRecord objects, use
```

Comment:
In the docs, can you clarify to the reader why we are using this? What are we using it in place of?

Reply:
I wrote "in place of `present?`", but I can make it more clear that we should use the other methods when dealing with activerecord objects. Mainly because the SQL queries involved are universally slower when using `present?` as opposed to the other methods